ext/opcache/ZendAccelerator.c

```diff
@@ -3755,6 +3755,19 @@ static zend_result preload_resolve_deps(preload_error *error, const zend_class_e
 	return SUCCESS;
 }
 
+static zend_result preload_update_constant(zval *val, zend_class_entry *scope)
+{
+	zval tmp;
+	ZVAL_COPY(&tmp, val);
+	if (zval_update_constant_ex(&tmp, scope) == FAILURE || Z_COLLECTABLE(tmp)) {
```

Comment:
I don't see why all ``COLLECTABLE`` values can't be resolved.
This disables resolution of all constants which values are arrays.

Reply:
Too conservative maybe yes, an alternative is looping (recursively) over the array and checking if they contain objects.